src/modules/ekf2/EKF/control.cpp

```diff
@@ -588,7 +588,7 @@ void Ekf::controlBetaFusion()
 			// activate the wind states
 			_control_status.flags.wind = true;
 			// reset the timeout timers to prevent repeated resets
-			_aid_src_sideslip.time_last_fuse = _imu_sample_delayed.time_us;
+			_aid_src_sideslip.time_last_fuse = imu_delayed.time_us;
```

Comment:
why not using `_time_delayed_us`?

Reply:
I was thinking we could make it clear IMU is required as control data, but maybe it's not worth the churn.